keyboards/ducky/one2mini/keymaps/ansi/keymap.c

```diff
@@ -0,0 +1,85 @@
+/* Copyright 2019 /u/KeepItUnder
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// LAYERS
+#define _QWERTY 0       // Standard QWERTY layer
+#define _QW _QWERTY
+#define _FUNCTION 1     // Function key layer
+#define _FN _FUNCTION
+#define _COLOUR 2       // RGB key layer
+#define _CLR _COLOUR
+
+#define MO_FN MO(1)
```

Comment:
You don't seem to be using these defines in any of the three keymaps here, so they should be removed.

Reply:
I'm assuming you mean the two MO_x defines (the tope two are used in the _FUNCTION layer)? I'll remove the other two.

Reply:
Yep, just these two here. I can see you're using `MO(_FN)` etc. in the keymaps themselves.